python/graphstorm/gsf.py

```diff
@@ -233,6 +235,28 @@ def create_builtin_edge_model(g, config, train_task):
                                      num_classes,
                                      multilabel=config.multilabel,
                                      target_etype=target_etype)
+        elif decoder_type == "MLPEFeatEdgeDecoder":
```

Comment:
why is it called `MLPEFeatEdgeDecoder`? MLP edge feature edge decoder?

Reply:
MLP with edge feature edge decoder.